### ISSUE TITLE
Which loss function is used in im2text?

### ISSUE THREAD

**Issue by mrgloom** [OpenNMT/OpenNMT-py#1797]
Which loss function is used in im2text?
https://opennmt.net/OpenNMT-py/im2text.html

**Comment by da03** [comment]
It's cross-entropy loss, same as in other sequence prediction tasks such as translation.

**Comment by mrgloom** [comment]
I wonder what are pros and cons of `cnn + rnn + ctc loss` compared to `seq2seq + cross-entropy loss` models? Is `seq2seq` a more modern type of architecture?
